crawl-ref/source/dat/des/branches/zot.des

```diff
@@ -2786,6 +2786,41 @@ MAP
 ............?............
 ENDMAP
 
+NAME:   dreamdust_zin_vs_xom
+TAGS:   no_monster_gen
+ORIENT: float
+DEPTH:  Zot, !Zot:$
```

Comment:
Not actually necessary to exclude Zot:$, as we allow random vaults on that level unless they somehow couldn't function properly. It even places killer klowns, which a monster the often only first encounters on Zot:5 itself.